x11/luit/Portfile

```diff
@@ -1,31 +1,31 @@
 PortSystem          1.0
 
 name                luit
-version             1.1.1
+version             2.0.20220111
 categories          x11
 license             X11
 platforms           darwin
-maintainers         {jeremyhu @jeremyhu} openmaintainer
-description         X.org luit
-long_description    Convert terminal i/o from legacy encodings to UTF-8
+maintainers         {invisible-island.net:dickey @ThomasDickey} openmaintainer
+description         filter that converts legacy encodings to/from UTF-8
+long_description    ${name} is a filter that converts terminal I/O \
+                    between legacy encodings and UTF-8.
 
-homepage            https://www.x.org/
-master_sites        xorg:individual/app/
+homepage            https://invisible-island.net/${name}/
+master_sites        https://invisible-mirror.net/archives/${name}/current/
 
-checksums           sha1    3130c14d7267cecce0ba2280643844b48cca49b0 \
-                    rmd160  571ec95ef3be0f761810e50272071ed4273afc16 \
-                    sha256  30b0e787cb07a0f504b70f1d6123930522111ce9d4276f6683a69b322b49c636
+extract.suffix      .tgz
+
+checksums           sha1    6febf78223c4afd7e110e0daf8e0cd92535e783e \
```

Comment:
`sha1` checksums are deprecated and may be dropped.

The recommended checksum types are `rmd160`, `sha256`, & `size`.